lib/Echidna/Types/Campaign.hs

```diff
@@ -84,6 +84,8 @@ data WorkerState = WorkerState
   , runningThreads :: [ThreadId]
     -- ^ Extra threads currently being run,
     --   aside from the main worker thread
+  , lastShrinkP :: Maybe Int
```

Comment:
Do you really need this field? Can you calculate it each time from the reproducer size?